main.py

```diff
@@ -1,53 +1,58 @@
-import cv2, numpy as np, time, sys
+import time
+import sys
 
-config = "yolov3.cfg"
-weights = "yolov3.weights"
+import cv2
+import numpy as np
 
-with open("coco.names") as f:
+CONFIDENCE = 0.5
+CONFIG = "yolov3.cfg"
+WEIGHTS = "yolov3.weights"
+LABELS = "coco.names"
```

Comment:
@guinnesschen Constants should be named in all capital letters. Please refer to PEP8 (https://www.python.org/dev/peps/pep-0008/#constants)